### DISCUSSION TITLE
CP issue? No one can hear my QSO

### DISCUSSION THREAD

**Discussion by klaute** [hmatuschek/qdmr#421] [Q&A]
Hello,

I’m starting again to learn how to use DMR in the German Brandmeister.network.

About 8 years ago I already owned a md380 gps dmr radio but sold it for some reason.

Now I own a Retevis RT3S. I have installed qdmr on my Mac and already created a Codeplug as I remember how it is done.

But I’m facing an issue which I can’t identify the reason for. So I’m not sure if my configuration is right.

I do receive QSO‘s from all configured DMR repeater channels, and TH’s like TG 262, 8, and so on.

But I can’t do an QSO, no one can hear me. I tried to call the echo 9990, the parrot and also different groups. No success. Only get the message „unknown id“, which confuses me because I do have my personal DMR ID for about 8 years.

I have attached my latest Yaml file.

Hopefully my issue is not only a dumb newbie problem, and also hope that it is fine to place the question at this place.

I would appreciate any suggestions.

Thank you!

—-

My radio firmware: P019.006
CP ver.: 01.12
QDMR version: 0.11.3 (Macports)

—-

**Comment by klaute** [comment]
This is my latest file:

[20240324_rt3s.yaml.txt](https://github.com/hmatuschek/qdmr/files/14739647/20240324_rt3s.yaml.txt)

**Comment by klaute** [comment]
I missed to note that if I press PTT (only on TG’s) I can see my call in Brandmeister network last active list. If I do that on a „private call“ entry like 9990 (echo) nothing happens. Only the led blinks red and green.

The same happens if I press PTT in the manual call menu.

**Comment by klaute** [comment]
I had some success yesterday. I can't reproduce it but I was able to do a PC to the Echo (9990) a few times. But I heard my voice only once.

![image](https://github.com/hmatuschek/qdmr/assets/352901/cc3530a5-57a0-4a37-a0e1-86b10888ac12)

For comparison:

![image](https://github.com/hmatuschek/qdmr/assets/352901/72abdf12-2002-4179-9d23-53b390eae8f0)

**Comment by klaute** [comment]
I tested my connection to TG98 while listening to https://hose.brandmeister.network/ , I successfully can hear my voice.

![image](https://github.com/hmatuschek/qdmr/assets/352901/82f49924-8d68-4848-aaca-54323bcff1a3)

**Comment by allesand** [comment]
> I missed to note that if I press PTT (only on TG’s) I can see my call in Brandmeister network last active list. If I do that on a „private call“ entry like 9990 (echo) nothing happens. Only the led blinks red and green.
> 

Is there a setting in the CPS to allow or deny private calls? Side note: There is OpenGD77 for the RT3S, which is very different from the stock firmware, more ham-friendly. No SMS support but much more comfortable to use for ham use.
But: OpenGD77 support for the RT3S in QDMR is not working properly at the moment, so if you want to use QDMR, it only works with the stock firmware currently (see #394)

**Comment by klaute** [comment]
The advice to look for settings which could prevent private calls was right, in the qDMR documentation it is noticed that for ham radio the setting "private call confirmed" needs to be disabled.

For every single channel.

I checked in my settings on my git server: https://git.okoyono.de/klaute/Retevis_RT3S_Codeplug

The "uknown id" issue still appears but only for my owm ID so that's a minor topic.

**Comment by allesand** [comment]
This is the mentioned part in the documentation? 

https://dm3mat.darc.de/qdmr/manual/ch04s03.html

**Reply by klaute** [reply]
This one: https://dm3mat.darc.de/qdmr/manual.pdf
